schedule/functional/allpatterns.yaml

```diff
@@ -1,13 +1,24 @@
 name:           allpatterns
 description:    >
-    Maintainer: slindomansilla.
+    Maintainer: foursixnine.
     Installation with all patterns selected for installation to check for
     potential package conflicts, how the system handles big space usage, etc.
     allpatterns installations can take longer, especially on non-x86_64
-    architectures.
+    architectures. This test suite doesn't publish an HDD, and doesn't run
+    post-installation tests.
```

Comment:
ok, after deleting the other YAML, I think we have to add here info about the difference between all patterns from default SLE modules and from selected compatible SLE modules.

Could you add a mention like
```
All SLE modules that are compatible between them can be selected by adding the setting ENABLE_ALL_SCC_MODULES=1
```

Reply:
@SergioAtSUSE Done